### PR TITLE
Page Titles Enhancement

### PR DESCRIPTION
**Pull request by Sinan997**
image
image

### CODE REVIEW

File: npm/ng-packs/packages/core/src/lib/services/title-strategy.service.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    constructor() {
+        super();
+        effect(() => {
+            if (this.langugageChange()) {
+                this.updateTitle(this.routerState);
+            }
+        });
+    }
+}
+// .pipe(tap(() => this.updateTitle(this.routerState))).subscribe();
```
Comment (masum-ulu) [inline npm/ng-packs/packages/core/src/lib/services/title-strategy.service.ts]: Really :) ?

Comment (Sinan997) [reply npm/ng-packs/packages/core/src/lib/services/title-strategy.service.ts]: i must remove comment lines 😄

**Comment by masum-ulu** [comment]
I'm checking on my local wait for approvment

**Review by masum-ulu — APPROVED** [review]

**Review by oykuermann — CHANGES_REQUESTED** [review]
I'll check after changes: https://github.com/volosoft/volo/pull/16718#issuecomment-1943812163

**Review by oykuermann — APPROVED** [review]

**Pull request merged**